README.md

```diff
@@ -152,16 +152,17 @@ The base watcher is useful in situations where you only want to use the servers
 It has the following options:
 
 * `method`: base
-* `label_filter`: optional filter to be applied to discovered service nodes
+* `label_filters`: optional list of filters to be applied to discovered service nodes
```

Comment:
What do you think about leaving label_filter in there for backwards compatibility. I know technically we're in 0.X releases here, but it's pretty easy to maintain backwards compatibility here (just accept both, append `label_filter` to `label_filters`.